src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -227,6 +226,12 @@ private void validateMedicalInventory(MedicalInventory medInventory) throws OHDa
 		}
 		if (reference == null || reference.equals("")) {
 			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.inventory.mustenterareference.msg")));
+		} else {
+			boolean exist = movStockInsertingManager.refNoExists(reference);
```

Comment:
yes for the moment

Reply:
so you have to check if that reference exists together with the suffixes, otherwise when it is time to create the actual movements will say that the reference already exists

Reply:
okay thank for that precision